common/core/web/keyboard-processor/src/text/kbdInterface.ts

```diff
@@ -457,7 +465,7 @@ namespace com.keyman.text {
                   deadContext[i].set();
                 }
                 // 'n' for 'notany'.  If we actually match or if we have nul context (\uFFFE), notany fails.
-              } else if(r.n && (result || context[i] !== NUL_CONTEXT)) {
+              } else if(r.n && (result || context[i] === NUL_CONTEXT)) {
```

Comment:
This test seemed to be backwards. It could be dealt with as a separate PR because it's a separate bug, but I'm reasonably comfortable with rolling it in with this fix.

I feel like this change needs a unit test, but wanted to put this in front of you first.

Reply:
Seems fine to me; it's related enough.